store/types/pruning.go

```diff
@@ -1,35 +1,66 @@
 package types
 
-// PruningStrategy specifies how old states will be deleted over time where
-// keepRecent can be used with keepEvery to create a pruning "strategy".
+var (
+	// PruneEverything defines a pruning strategy where all committed states will
+	// be deleted, persisting only the current state.
+	PruneEverything = PruningOptions{
+		KeepEvery:     1,
```

Comment:
Interesting with this option, you will have significantly more IO than `PruneSyncable`, yet much less disk usage.

What about this just keeping one flushed to disk (as now), but only flushing every 10 blocks or so? (just increment `KeepEvery`). This would be helpful for nodes that need throughput

Reply:
I would prefer to keep sound and most "general" strategies as we have now, except allow the CLI/start command to accept manual values for these instead of providing a bunch of other strategies. ie. `gaiad start --pruning-keep-every ... --pruning-snapshot-every ...`

Reply:
Actually probably best to set these in `app.toml` instead of (or in additon to) the cli.

But I agree with your point, leaving these 3 safe named posibilities, and then allowing the user to take the risk and shoot themself in the foot if they want to do advanced configuration